enterprise/frontend/src/metabase-enterprise/auth/index.js

```diff
@@ -38,13 +38,15 @@ PLUGIN_ADMIN_SETTINGS_UPDATES.push(sections =>
         noHeader: true,
         widget: SamlAuthCard,
         getHidden: () => !hasPremiumFeature("sso_saml"),
+        forceRenderWidget: true,
```

Comment:
I'm not following what `forceRenderWidget` is accomplishing?

Reply:
Originally, if `setting.is_env_setting` was set to true, we should not render a widget, only some text with a message. I made a change in `SettingsSetting.jsx` to check this new prop. If it's set to true, we'll render the widget defined in the setting definition regardless, and it'll be the widgets responsibility to handle the env var case

Reply:
for the record, I hate this, but can't think of a better way